src/client.js

```diff
@@ -0,0 +1,30 @@
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Client {
+  constructor (socket) {
+    this.socket = socket
+  }
+
+  #modifiedTransaction (tx) {
```

Comment:
I think that this function is not really related to the Client, I would put it in the utils or better in some other class method function.

Actually, I think it could be even better if we send message to the client as is and do not modify it. Then developer can easily rely on currently existing dashcore-lib documentation (since its already typed) to work with your data.

Reply:
I have seen websocket services structured like that, so I assumed we should do something similar. However, you’re right developers who will  use it can modify it themselves.